--- aqute-bnd-0.0.363/src/main/java/aQute/bnd/ant/WrapTask.java.orig	2013-12-18 11:46:52.390519630 -0200
+++ aqute-bnd-0.0.363/src/main/java/aQute/bnd/ant/WrapTask.java	2013-12-18 11:51:56.197510331 -0200
@@ -42,7 +42,8 @@ public class WrapTask extends BaseTask i
 				String name = file.getName();
 				name = name.replaceFirst("(\\.jar)?$", ".bnd");
 				File bndFile = new File(definitions, name );
-				bnd.doWrap(bndFile.exists()?bndFile:null, file, output, null, 0, getProject().getProperties());
+				Map<String,String> map = new HashMap(getProject().getProperties());
+				bnd.doWrap(bndFile.exists()?bndFile:null, file, output, null, 0, map);
 				failed |= report(bnd);
 			}
 		} catch (Exception e) {
